canonical-websites/snapcraft.io
# Add 'unlisted' to snap privacy settings

Fixes #1559

Adds 'Unlisted' to privacy snap settings 

<img width="1030" alt="screenshot 2019-02-14 at 17 03 33" src="https://user-images.githubusercontent.com/83575/52799614-7a72d480-307a-11e9-9f23-fa76c07a0ff8.png">

### QA
- ./run or https://snapcraft-io-canonical-websites-pr-1590.run.demo.haus/
- go to Settings page of any snap you own
- privacy settings should have a new 'Unlisted' option
- change privacy to unlisted
- snap should be public (details page available) but will dissapear from search results - this may take a minute or so to happen
- change privacy back to public or private
- all should work as expected
